examples/ASHRAE 211 Export.xml

```diff
@@ -426,7 +426,7 @@
               <HeatingSources>
                 <HeatingSource ID="HeatingSource-a46904f7-c076-421a-beb1-e909a3d81aa4">
                   <HeatingSourceType>
-                    <SourceHeatingPlantID IDref="HeatiingPlant1"/>
+                    <auc:HeatingPlantID xmlns:auc="http://buildingsync.net/schemas/bedes-auc/2019" IDref="HeatiingPlant1"/>
```

Comment:
why is the XML namespace in this element? Shouldn't it just have the `IDref` attribute?

Reply:
probably an artifact of the XSLT and the doc using a default namespace? I'm not sure how to fix this easily... might need to be manual?

Reply:
This is a pain point for me as well. It comes from the translator, which tries to add namespace `auc` for changed elements and declare (again) the namespace directory. If we don't want this functionality, then it'll not add `auc` for changed elements in those example files with namespace in all elements, which again will make a lots of outstanding differences in other examples.
So far, I can't think of an automatic way to identify the two different patterns in all xml examples in xslt, but I can definitely manually change them.

Reply:
I'm thinking we will need to go through and manually update. I don't like having the local namespace, especially in the example files.

Reply:
Will do!

Reply:
Done